Clear-ComputeNode.ps1

```diff
@@ -0,0 +1,233 @@
+Param (
+    [Parameter(Mandatory = $false)] [String] $AdapterName = "Ethernet1",
+    [Parameter(Mandatory = $false)] [String] $ForwardingExtensionName = "vRouter forwarding extension",
+    [Parameter(Mandatory = $false)] [String] $VMSwitchName = "Layered Ethernet1",
+    [Parameter(Mandatory = $false)] [String] $ConfigAndLogDir = "C:\ProgramData\Contrail",
+    [Parameter(Mandatory = $false)] [String] $InstallationDir = "C:\Program Files\Juniper Networks"
+)
+
+function Stop-ProcessIfExists {
+    Param ([Parameter(Mandatory = $true)] [string] $ProcessName)
+
+    $Proc = Get-Process $ProcessName -ErrorAction SilentlyContinue
+    if ($Proc) {
+        $Proc | Stop-Process -Force -PassThru | Wait-Process -ErrorAction Stop
+    }
+}
+
+function Remove-HNSNetworks {
+    Write-Host "Cleaning HNS state..."
+    try {
+        Get-NetNat | Remove-NetNat -Confirm:$false
+        # Two tries are intentional - it's workaround for HNS behavior.
+        Get-ContainerNetwork | Remove-ContainerNetwork -ErrorAction SilentlyContinue -Force
+        Get-ContainerNetwork | Remove-ContainerNetwork -ErrorAction Stop -Force
+    } catch {
+        $ErrorMessage += $_.Exception.Message
+        Write-Host "    This step failed with the following error message: $ErrorMessage"
+        Write-Host "    Trying to continue, but final result is unpredictable."
+    }
+}
+
+function Remove-NodeMgrService {
+    Write-Host "Stopping Node Manager and removing service..."
+    try {
+        $NodeMgrServiceName = "contrail-vrouter-nodemgr"
+        $Service = Get-Service $NodeMgrServiceName -ErrorAction SilentlyContinue
+        if ($Service -ne $null) {
+            Stop-Service $NodeMgrServiceName -ErrorAction Stop
+            sc.exe delete $NodeMgrServiceName
+            if ($LASTEXITCODE -ne 0) {
+                throw "sc.exe failed to delete service."
+            }
+        }
+    } catch {
+        $ErrorMessage += $_.Exception.Message
+        Write-Host "    This step failed with the following error message: $ErrorMessage"
+        Write-Host "    Trying to continue, but final result is unpredictable."
+    }
+}
+
+function Remove-AgentService {
```

Comment:
All `Remove-*Service` functions are really similar. Maybe we can remove this triplication?